src/SUnit-Tests/SUnitTest.class.st

```diff
@@ -422,9 +443,11 @@ SUnitTest >> testGreenTestThenLongRunningTest [
 SUnitTest >> testHangedChildProcessTestWhenItIsAllowedToLeaveProcessesAfterTest [
 
 	| case result hangedProcess |
-
+	
 	case := self class selector: #hangedChildProcessTest.	
 	case executionProcessMonitor allowTestToLeaveProcesses.
+	forkedProcesses := OrderedCollection new.
+	case forkedProcesses: forkedProcesses.
```

Comment:
Would be nice to not repeat the code between this, `testHangedChildProcessTestWhenLeftProcessIsConsideredAsFailure`,`testHangedTestDueToFailedChildProcess`, `testFailedTestWithFailedChildProcessTestShouldNotEffectFollowingAno` and `testFailedChildProcessTest`